app/src/main/java/org/wikipedia/compose/components/error/ComposeWikiErrorView.kt

```diff
@@ -146,12 +148,109 @@ fun ComposeWikiErrorView(
 
 @Preview
 @Composable
-private fun ComposeWikiErrorParentViewPreview() {
+private fun WikiErrorViewPreview() {
     BaseTheme(
         currentTheme = Theme.DARK
     ) {
-        ComposeWikiErrorParentView(
+        WikiErrorView(
             caught = Exception()
         )
     }
 }
+
```

Comment:
Would it be easier to find if we move the `@Preview` function to the bottom of the class?